### ISSUE TITLE
Is the pointer zoom feature not working?

### ISSUE THREAD

**Issue by vingrad** [scaleflex/js-cloudimage-360-view#159]
I've observed that the zoom functionality of the pointer doesn't seem to operate correctly. I attempted to use it within my application, and I also tested it on the official demo located at https://scaleflex.github.io/js-cloudimage-360-view/

However, I encountered the following error:

```
load-image-as-promise.js:5 Uncaught TypeError: t is not a function
    at Image.n (load-image-as-promise.js:5:29)
```

**Comment by eminozlem** [comment]
Yes it seems "pointer-zoom" stopped working. I'd also like to get a fix on it.
It seems its broken since v3.1.0 , its fine at 3.0.4
But the most important fix for me was the "vertical scroll" fix at 3.1.0. So I am stuck on which version to use. 
I hope this gets fixed soon

**Comment by eminozlem** [comment]
Ok so this is the culprit which fixes the y-scroll but breaks the pointer zoom

![image](https://github.com/scaleflex/js-cloudimage-360-view/assets/507708/9f73c5af-ae07-4f08-844f-a1026d281f7d)
